homie-controller/src/types.rs

```diff
@@ -157,9 +165,16 @@ pub struct Property {
 
     /// The format of the property, if any. This should be specified if the datatype is `Enum` or
     /// `Color`, and may be specified if the datatype is `Integer` or `Float`.
+    ///
+    /// This field holds the raw string received from the device; use
```

Comment:
; is distracting
```suggestion
    /// This field holds the raw string received from the device. Use
```
(same below)

Reply:
Done.